Initialize-Server.ps1

```diff
@@ -23,7 +23,7 @@ Set-StrictMode -Version 'Latest'
 $PSCommandPath = $MyInvocation.MyCommand.Definition
 $PSScriptRoot = Split-Path -Parent -Path $PSCommandPath
 
-$os = Get-Cim -Class 'Win32_OperatingSystem'
+$os = Get-CCimInstance -Class 'Win32_OperatingSystem'

```

Comment:
This isn't going to work because `Get-CCimInstance` is a private function. We can't run anything on Windows 2008 anymore, so go ahead and remove lines 26 through 37 and line 39. Leave the `Install-WindowsFeature` line.